tools/sanity_checks.py

```diff
@@ -211,14 +192,10 @@ def setUpClass(cls):
 
         try:
             cls.releases = Releases.load()
-            fn = 'ci_config.json'
-            with open(fn, 'r', encoding='utf-8') as f:
-                cls.ci_config = json.load(f)
+            cls.ci_config = CIConfig.load()
         except json.decoder.JSONDecodeError as ex:
             raise RuntimeError('metadata is malformed') from ex
 
-        system = platform.system().lower()
-        cls.skip = T.cast(T.List[str], cls.ci_config[f'broken_{system}'])
```

Comment:
First argument to T.cast should always be quoted. Apparently it wasn't. Still, we can fix that while we're moving this around...